### ISSUE TITLE
Error in New Relationship for Fresh Install

### ISSUE THREAD

**Issue by rafny** [monicahq/monica#3035]
After a fresh install, I logged in as admin@admin.com and created 2 contacts. Then when I go to either contact page and click on "Add Love Relationship" or "Add Family Relationships" I get the same SQL error:
`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'contacts.account_id' in 'where clause' (SQL: select count(*) as aggregate from `frBcontacts` where `contacts`.`account_id` = 1 AND (`contacts`.`first_name` LIKE '%%' or `contacts`.`middle_name` LIKE '%%' or `contacts`.`last_name` LIKE '%%' or `contacts`.`nickname` LIKE '%%' or `contacts`.`description` LIKE '%%' or `contacts`.`job` LIKE '%%') AND `id` != 1)`

Note: frB is the table prefix.
Error screenshot: https://postimg.cc/VJs8YzsL
```
mysql> describe frBcontacts;
+------------------------------+------------------+------+-----+---------+----------------+
| Field                        | Type             | Null | Key | Default | Extra          |
+------------------------------+------------------+------+-----+---------+----------------+
| id                           | int(10) unsigned | NO   | PRI | NULL    | auto_increment |
| account_id                   | int(11)          | NO   | MUL | NULL    |                |
| first_name                   | varchar(255)     | NO   |     | NULL    |                |
| middle_name                  | varchar(255)     | YES  |     | NULL    |                |
| last_name                    | varchar(255)     | YES  |     | NULL    |                |
| nickname                     | varchar(255)     | YES  |     | NULL    |                |
| gender_id                    | int(11)          | YES  |     | NULL    |                |
| description                  | varchar(255)     | YES  |     | NULL    |                |
| uuid                         | char(36)         | YES  |     | NULL    |                |
| is_starred                   | tinyint(1)       | NO   |     | 0       |                |
| is_partial                   | tinyint(1)       | NO   |     | 0       |                |
| is_active                    | tinyint(1)       | NO   |     | 1       |                |
| is_dead                      | tinyint(1)       | NO   |     | 0       |                |
| deceased_special_date_id     | int(10) unsigned | YES  | MUL | NULL    |                |
| deceased_reminder_id         | int(10) unsigned | YES  | MUL | NULL    |                |
| last_talked_to               | date             | YES  |     | NULL    |                |
| stay_in_touch_frequency      | int(11)          | YES  |     | NULL    |                |
| stay_in_touch_trigger_date   | datetime         | YES  |     | NULL    |                |
| birthday_special_date_id     | int(10) unsigned | YES  | MUL | NULL    |                |
| birthday_reminder_id         | int(10) unsigned | YES  | MUL | NULL    |                |
| first_met_through_contact_id | int(11)          | YES  |     | NULL    |                |
| first_met_special_date_id    | int(10) unsigned | YES  | MUL | NULL    |                |
| first_met_reminder_id        | int(10) unsigned | YES  | MUL | NULL    |                |
| first_met_where              | varchar(255)     | YES  |     | NULL    |                |
| first_met_additional_info    | longtext         | YES  |     | NULL    |                |
| job                          | varchar(255)     | YES  |     | NULL    |                |
| company                      | varchar(255)     | YES  |     | NULL    |                |
| food_preferences             | longtext         | YES  |     | NULL    |                |
| avatar_source                | varchar(255)     | NO   |     | default |                |
| avatar_gravatar_url          | varchar(250)     | YES  |     | NULL    |                |
| avatar_adorable_uuid         | char(36)         | YES  |     | NULL    |                |
| avatar_adorable_url          | varchar(250)     | YES  |     | NULL    |                |
| avatar_default_url           | varchar(250)     | YES  |     | NULL    |                |
| avatar_photo_id              | int(11)          | YES  |     | NULL    |                |
| has_avatar                   | tinyint(1)       | NO   |     | 0       |                |
| avatar_external_url          | varchar(400)     | YES  |     | NULL    |                |
| avatar_file_name             | varchar(255)     | YES  |     | NULL    |                |
| avatar_location              | varchar(255)     | NO   |     | local   |                |
| gravatar_url                 | varchar(255)     | YES  |     | NULL    |                |
| last_consulted_at            | timestamp        | YES  |     | NULL    |                |
| number_of_views              | int(11)          | NO   |     | 0       |                |
| created_at                   | timestamp        | YES  |     | NULL    |                |
| updated_at                   | timestamp        | YES  |     | NULL    |                |
| default_avatar_color         | varchar(255)     | NO   |     | NULL    |                |
| has_avatar_bool              | tinyint(1)       | NO   |     | 0       |                |
+------------------------------+------------------+------+-----+---------+----------------+
```

```
mysql> select * from frBcontacts;
+----+------------+------------+-------------+-----------+----------+-----------+-------------+--------------------------------------+------------+------------+-----------+---------+--------------------------+----------------------+----------------+-------------------------+----------------------------+--------------------------+----------------------+------------------------------+---------------------------+-----------------------+-----------------+---------------------------+------+---------+------------------+---------------+---------------------+--------------------------------------+------------------------------------------------------------------------------+-------------------------------------------------------------+-----------------+------------+---------------------+------------------+-----------------+--------------+---------------------+-----------------+---------------------+---------------------+----------------------+-----------------+
| id | account_id | first_name | middle_name | last_name | nickname | gender_id | description | uuid                                 | is_starred | is_partial | is_active | is_dead | deceased_special_date_id | deceased_reminder_id | last_talked_to | stay_in_touch_frequency | stay_in_touch_trigger_date | birthday_special_date_id | birthday_reminder_id | first_met_through_contact_id | first_met_special_date_id | first_met_reminder_id | first_met_where | first_met_additional_info | job  | company | food_preferences | avatar_source | avatar_gravatar_url | avatar_adorable_uuid                 | avatar_adorable_url                                                          | avatar_default_url                                          | avatar_photo_id | has_avatar | avatar_external_url | avatar_file_name | avatar_location | gravatar_url | last_consulted_at   | number_of_views | created_at          | updated_at          | default_avatar_color | has_avatar_bool |
+----+------------+------------+-------------+-----------+----------+-----------+-------------+--------------------------------------+------------+------------+-----------+---------+--------------------------+----------------------+----------------+-------------------------+----------------------------+--------------------------+----------------------+------------------------------+---------------------------+-----------------------+-----------------+---------------------------+------+---------+------------------+---------------+---------------------+--------------------------------------+------------------------------------------------------------------------------+-------------------------------------------------------------+-----------------+------------+---------------------+------------------+-----------------+--------------+---------------------+-----------------+---------------------+---------------------+----------------------+-----------------+
|  1 |          1 | Keri       | NULL        | LastName  | NULL     |         2 | NULL        | 0e80cdbb-44a3-469d-814c-d8c3be622db1 |          0 |          0 |         1 |       0 |                     NULL |                 NULL | NULL           |                    NULL | NULL                       |                     NULL |                 NULL |                         NULL |                      NULL |                  NULL | NULL            | NULL                      | NULL | NULL    | NULL             | default       | NULL                | 8dbeeece-7a3c-4639-8430-8f70c1926fa7 | https://api.adorable.io/avatars/200/8dbeeece-7a3c-4639-8430-8f70c1926fa7.png | avatars/0e80cdbb-44a3-469d-814c-d8c3be622db1.jpg?1569198714 |            NULL |          0 | NULL                | NULL             | local           | NULL         | 2019-09-23 02:08:48 |               5 | 2019-09-23 00:31:54 | 2019-09-23 00:31:54 | #bd5067              |               0 |
|  2 |          1 | Hydia      | NULL        | NULL      | NULL     |         2 | NULL        | f25b9f41-3376-45ef-b735-84e848961f68 |          0 |          0 |         1 |       0 |                     NULL |                 NULL | NULL           |                    NULL | NULL                       |                     NULL |                 NULL |                         NULL |                         1 |                  NULL | NULL            | Lilly's Mom               | NULL | NULL    | NULL             | default       | NULL                | 40bf2458-27f6-4aac-b6a5-2ffe60cd0f37 | https://api.adorable.io/avatars/200/40bf2458-27f6-4aac-b6a5-2ffe60cd0f37.png | avatars/f25b9f41-3376-45ef-b735-84e848961f68.jpg?1569199002 |            NULL |          0 | NULL                | NULL             | local           | NULL         | 2019-09-23 02:12:35 |               3 | 2019-09-23 00:36:42 | 2019-09-23 00:37:14 | #93521e              |               0 |
+----+------------+------------+-------------+-----------+----------+-----------+-------------+--------------------------------------+------------+------------+-----------+---------+--------------------------+----------------------+----------------+-------------------------+----------------------------+--------------------------+----------------------+------------------------------+---------------------------+-----------------------+-----------------+---------------------------+------+---------+------------------+---------------+---------------------+--------------------------------------+------------------------------------------------------------------------------+-------------------------------------------------------------+-----------------+------------+---------------------+------------------+-----------------+--------------+---------------------+-----------------+---------------------+---------------------+----------------------+-----------------+
```

Hydia URL: https://domain.com/monica/public/people/h:E51eWDRYQlO32r7p5Xjy9KPdNo6bkAMB

Monica 2.15.0
Self-hosted on AWS EC3

**Comment by rafny** [comment]
I just did a fresh reinstall without a database prefix, and did not get the error. Perhaps the statement needs to be updated to account for the prefix?

**Comment by asbiin** [comment]
This is now fixed in 2.15.1
